editable-templates/mvp-reference-pages/table-component.html

```diff
@@ -0,0 +1,69 @@
+---
+title: Table component
+hide_breadcrumb: false
+date_modified: 2024-10-31
+layout: no-container
+---
+
+<div class="">
+	<div class="container">
+		<div class="">
+			<div class="row">
+				<div class="title col-lg-12 col-xs-12">
+					<h1 property="name" id="wb-cont" dir="ltr"> Table component
+					</h1>
+				</div>
+				<div class="table-responsive col-lg-12">
+					<table class="table table-bordered">
+						<tbody>
+							<tr>
+								<td class="text-center"><strong>R1C1</strong></td>
+								<td class="text-center"><strong>R1C2</strong></td>
+								<td class="text-center"><strong>R1C3 <sup>NOTE</sup></strong></td>
+							</tr>
+							<tr>
+								<td>R2C1</td>
+								<td>R2C2</td>
+								<td><a title="Alt text"
```

Comment:
We should not allow "title" attribute on anchor.  Like only allow that if it is explicitly added by the web author like via the embed-html component.

```suggestion
								<td><a 
```